### ISSUE TITLE
Installer cannot be used

### ISSUE THREAD

**Issue by Palace4Software** [Palace4Software/timesync#4]
The installer (install-timesync-1.1.0.sh) cannot be used if the execution path is different.

Reproduction:
1. Download installer and ressources (save in ~/Downloads)
2. Start terminal
3. Type ``` chmod +x ./Downloads/install-timesync-1.1.0.sh ```
4. Type ``` sudo ./Downloads/install-timesync-1.1.0.sh ```

Output:
```
$ sudo ./Downloads/install-timesync-1.1.0.sh 

Checking if timesync is already installed...
Timesync is not installed. Checking resources archive...
The ressources archive cannot be found. Please be sure, that the file is called 'ts-ressources-1.1.0' and that you don't deleted it.
```

**Comment by Palace4Software** [comment]
Can be solved with:

1. ``` cd Downloads ```
2. ``` chmod +x ./Downloads/install-timesync-1.1.0.sh ```
3. ``` sudo ./install-timesync-1.1.0.sh ```

More comming soon.

**Comment by Palace4Software** [comment]
Fixed in 2.0.0
